site/docs/types/2/0215-software-components.md

```diff
@@ -0,0 +1,36 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project. -->
+
+# 0215 Software Components
+
+*DeployedSoftwareComponent* describes a code asset that is deployed to implement a [software capability](0042-software-server-capabilities.md). Each software component has a well defined interface describe by an [APISchema](0536-api-schemas.md) that is linked to the DeployedSoftwareComponent by the [AssetSchemaType](0503-asset-schema.md) relationship.
+
+*DeployedConnector* represents specialist software component called a *connector* that provides pluggable access to third party technologies.  These connectors implement the [Open Connector Framework (OCF)](../../../open-metadata-implementation/frameworks/open-connector-framework) interfaces.
```

Comment:
Right Link?

Reply:
Nope, the final link should be pointing to `/egeria-docs/frameworks/ocf`